mzLib/MassSpectrometry/MsDataFile.cs

```diff
@@ -149,26 +168,24 @@ public virtual IEnumerable<MsDataScan> GetMsScansInTimeRange(double firstRT, dou
         public virtual int GetClosestOneBasedSpectrumNumber(double retentionTime)
```

Comment:
please add a comment that says that the closest could be later in time and also not associated with a specific MS2

Reply:
done